I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/theremake.rpy", line 104, in script
    python:
  File "game/theremake.rpy", line 106, in <module>
    bday = date2.strip # "07282018"
NameError: name 'date2' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/theremake.rpy", line 104, in script
    python:
  File "/home/teppy/Downloads/renpy-7.4.11-sdk/renpy/ast.py", line 928, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/home/teppy/Downloads/renpy-7.4.11-sdk/renpy/python.py", line 2245, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/theremake.rpy", line 106, in <module>
    bday = date2.strip # "07282018"
NameError: name 'date2' is not defined

Linux-5.13.0-39-generic-x86_64-with-debian-bullseye-sid
Ren'Py 7.4.11.2266
DigiLet_pinky 1.0
Thu Apr 21 19:34:50 2022
